README.md

```diff
@@ -1,3 +1,223 @@
+
+[file system routing]: https://www.gatsbyjs.com/docs/reference/routing/file-system-route-api/
+
+### Theme updates
+
+You can choose to either leave this directory in your site, or remove it to install and use the published version of the theme from npm.
+If you install the theme from npm, your site can receive upstream updates and bug fixes in the future.
+
+## Installing the theme in an existing site
+
+Because this starter is built with a Gatsby theme, you can leverage its functionality in an existing site without cloning this starter.
+For more information, see the theme's [README.md][theme readme].
+
+## Adding a layout
+
+By default, the theme's landing pages do not include a wrapping layout. This is to allow you customize the header, footer and other wrapping content to match the rest of your site.
+This starter shadows the theme's layout with the `src/gatsby-theme-landing-page/components/layout.js` file, which renders the `src/components/layout.js` file. Edit this file to customize the shared layout for all landing pages.
+
+```js
+// example src/gatsby-theme-landing-page/components/layout.js
+export { default } from "../../components/custom-layout";
+```
+
+## Customizing the typography and colors
+
+To customize the built-in components' typography, colors, and layout, edit the `src/gatsby-theme-landing-page/styles/variables.module.css` file.
+
+```css
+/* example src/gatsby-theme-landing-page/styles/variables.module.css */
+.root {
+  /* typography */
+  --font: "Inter", sans-serif;
+  --font-heading: "Poppins", sans-serif;
+  --line-height: 1.5;
+  --font-size-1: 12px;
+  --font-size-2: 14px;
+  --font-size-3: 16px;
+  --font-size-4: 24px;
+  --font-size-5: 32px;
+  --font-size-6: 48px;
+  --letter-spacing-caps: 0.03em;
+  --font-weight-light: 300;
+  --font-weight-normal: 400;
+  --font-weight-bold: 700;
+  /* colors */
+  --text-color: black;
+  --text-color-secondary: #555;
+  --background-color: white;
+  --link-color: #07c;
+  --link-hover-color: #05a;
+  --primary-color: #08d;
+  --secondary-color: #70c;
+  --button-color: white;
+  --button-background-color: #07c;
+  --button-hover-color: #05a;
+  --button-secondary-color: #07c;
+  --button-secondary-background-color: white;
+  --button-secondary-hover-color: rgb(215, 232, 250);
+  /* layout */
+  --space-1: 4px;
+  --space-2: 8px;
+  --space-3: 16px;
+  --space-4: 32px;
+  --space-5: 64px;
+  --space-6: 128px;
+  --max-width: 1024px;
+  --max-width-narrow: 768px;
+  --border-radius: 4px;
+  /* shadows */
+  --box-shadow-raised: 0px 1px 2px rgba(46, 41, 51, 0.08), 0px 2px 4px rgba(71, 63, 79, 0.08);
+  --box-shadow-floating: 0px 2px 4px rgba(46, 41, 51, 0.08), 0px 4px 8px rgba(71, 63, 79, 0.16);
+}
+```
+
+## Customizing and extending components
+
+To customize a landing page section component, create a file in `src/gatsby-theme-landing-page/sections/` with the same name used in the theme. This will shadow the built-in component to completely override it.
+
```

Comment:
Might be helpful to include a screenshot here of what we're talking about, as it took me a moment to understand which part of Contentful this is referring to